rhel-observability-aws-cloudwatch.spec

```
@@ -24,6 +24,10 @@ mkdir -p %{buildroot}%{_unitdir}
 install -p -m 0644 -D %{SOURCE1} %{buildroot}%{_sysconfdir}/opentelemetry-collector-cloudwatch-config/config.yaml
 install -p -m 0644 -D %{SOURCE2} %{buildroot}%{_unitdir}/%{name}.service
 
+%pre
+/usr/bin/getent group aws-observability > /dev/null || /usr/sbin/groupadd -r aws-observability
+/usr/bin/getent passwd aws-observability > /dev/null || /usr/sbin/useradd -r -M -s /sbin/nologin -g aws-observability -G systemd-journal aws-observability
+
```

Consider adding a %postun routine to clean the user once the package is removed from the system
```suggestion
%postun
/usr/sbin/userdel aws-observability
```